### DISCUSSION TITLE
Common concepts of psychosecurity

### DISCUSSION THREAD

**Discussion by HyperCrowd** [PsySecGroup/foundation#3] [Ideas]
Psychosecurity is pretty specifically defined here.  What are some common words that people use to touch on psychosecurity concepts?

Examples:

* psychometrics
* ad targeting
* shills
* conspiracy
* censorship
* thoughtcrime

What else?

**Comment by AirFireWater** [comment]
- psychographics
- influence operations
- psychological lookalike modelling
- psychological operations
- nudge theory
- "A/B" testing
- OCEAN
- behavioural microtargeting

**Comment by PlainsDan** [comment]
discernment
"do your (own) research"

**Comment by HyperCrowd** [comment]
![psychohistory-pitch v1 0 min](https://user-images.githubusercontent.com/76200774/177060788-6577caf2-2476-49cd-9771-1cdb394fdabb.png)

First one-pager

**Comment by HyperCrowd** [comment]
From Nate

## Weaponized Distraction

Human attention is so easy to hijack. Consider in my previous post the differentiation between the sensor and intuitive personality types. With sensors making up 75% of the population, the majority of human action in a consumer society is oriented toward providing inputs for the senses.  This is why so many products are designed to stimulate the senses. The result is a lot of distracted living.

It seems that we have become a society of people who are chronically distracted. According to a recent study by the American Psychological Association, the average person checks their phone more than 80 times per day. We are bombarded with notifications, alerts, and calls for our attention every single day.

And’s not just our phones. We live in a world where we are constantly bombarded with stimuli. Televisions, radios, billboards, and even the people around us are vying for our attention. It’s no wonder that we have trouble focusing on anything for more than a few minutes at a time. 

Maslow's hierarchy identifies five basic needs for all humans: physiological, safety, love/belonging, esteem, and self-actualization. A closer look will reveal that four of these five needs are directly related to inputs for the senses. The need for physiological items such as food and water are related to the sense of taste and smell. The need for safety is often manifest in the form of a safe place to live, which provides inputs for the senses of sight and touch. Love and belonging include items such as companionship and intimacy, both of which provide inputs for the sense of touch. Finally, esteem needs such as power and status are often displayed through material possessions, which provide inputs for the senses of sight, touch, and sometimes sound. 

The seven deadly sins, as written about by early Christian theologians, are also rooted in inputs for the senses. The sin of lust, for example, is based on a strong desire for sexual gratification, which is an input for the sense of touch. The sin of gluttony is based on a desire for food and drink, which are inputs for the sense of taste and smell. 

It’s no coincidence that so many of our basic needs and desires are based on inputs for the senses. Our brains are wired to pay attention to sensory information. This is because our survival as a species depends on it. In the wild, things that stimulate our senses (e.g., bright colors, loud noises) are often signs of danger. Our brains have evolved to pay attention to these things to keep us safe.

The problem is that, in the modern world, there are so many things vying for our attention that it’s difficult to pay attention to anything for more than a few minutes at a time. Our brains are not evolved to deal with this level of stimulation, and as a result, we often end up feeling overwhelmed and stressed. Modern sales and marketing professionals are keenly aware of how to manipulate these needs and desires. Advertisers use sex, violence, and other sensational (appealing to the senses) topics to get our attention and then sell us things we don’t need. They know how to hijack our attention and use it to their advantage. 

It is a single step from commercialization to weaponization, and there is nothing more dangerous than a distraction. When we are constantly bombarded with stimuli, it’s difficult to focus on anything for more than a few minutes at a time. This makes it hard to pay attention to important things, like our relationships, our job, or our health. The constant stream of information can also lead to overwhelm and stress.

When population groups are targeted for distraction,  it can have disastrous consequences. For example, during the 2016 US presidential election, social media was used to disseminate false information and sow discord among the American people. The goal was to create so much chaos that people would be distracted from the real issues at hand. And it worked.

The weaponization of distraction is a serious problem that we need to address. We need to find ways to protect ourselves from the constant onslaught of stimuli and learn how to focus on what’s important. Otherwise, we run the risk of being controlled by those who know how to manipulate our attention.

There are three main types of distractions:

1. Sensory distractions: These are things that capture our attention through our senses (e.g., bright colors, loud noises).
1. Emotional distractions: These are things that play on our emotions (e.g., fear, anger, joy).
1. Cognitive distractions: These are things that require mental effort to process (e.g., complex problems, difficult tasks).

These vectors of attack can be used individually or in combination to create a powerful distraction. Advertisers, for example, often use a combination of sensory and emotional distractions to sell their products. And political campaigns have been known to use all three types of distractions to control the narrative and deflect attention from important issues.

The best way to protect ourselves from the weaponization of distraction is to become aware of how it works and learn how to counter it. We can start by paying attention to our own attention and learning how to focus on what’s important. This requires practice and effort, but it’s worth it. Because when we are in control of our attention, we are in control of our lives.

Focusing on identifying the mechanisms being used to distract provides the ability to identify a motive. If the motive is to sell a product, then the goal is to convince you to make a purchase. This can be countered by researching the product before making a decision. If the motive is to control the narrative, then the goal is to change your perspective. This can be countered by seeking out alternative sources of information. When you consider the strong reaction from traditional media outlets towards citizen journalism, it becomes clearer that the goal is to control. 

Becoming aware of these mechanisms is a direct attack on their means of attaining and retaining power. The motive is simple; captured attention is captured market share, and captured market share is converted into power. This power is used to strengthen influence and control. It is a self-reinforcing cycle. 

### Philosophical Mechanics

The deeper context of this mechanism lies in metaphysics. Metaphysics is the world of symbols.  These symbols are the things we use to represent our reality. Language is the most obvious example, but there are others, like money and laws. The way these symbols are used reflects and shapes our reality. 

A symbol is a condensed unit of information that has the ability to represent a complex reality. In order to create a symbol, we need to understand the complex reality that we want to represent. This is why language is so important. It is the tool that we use to create symbols. Language is a system of symbols, itself, that we use to communicate our thoughts and experiences. The words we use are like building blocks that we can assemble into sentences and paragraphs. These sentences and paragraphs can then be used to express our ideas.

The quality of our ideas is directly related to the quality of our language. This is why it’s so important to choose our words carefully. The words we use not only influence how others see us, but they also influence how we see ourselves.

For example, take the symbol of money. Money is a representation of value. It is a way of exchange that allows us to trade goods and services without having to barter. This system works because we all agree on the value of money. We trust that a dollar today will be worth a dollar tomorrow. But what if we didn’t agree on the value of money? What if some people thought a dollar was worth more than a dollar, and some people thought it was worth less? This would create chaos. People would start hoarding money, and the economy would collapse.

This is what happened in the lead-up to the financial crisis of 2008. Wall Street bankers created a system that they didn’t understand, and they convinced everyone else to play along. They did this by creating new symbols (e.g., subprime mortgages) and using them to represent value. This worked for a while, but eventually, the system collapsed because the underlying reality didn’t match the representation.

The same thing is happening with our attention. We are being inundated with new symbols (e.g., likes, shares, views) that are being used to represent our attention. And just like with money, we are agreeing to play along without understanding the consequences. These new symbols are being weaponized to distract us. They are being used to capture our attention and sell it to the highest bidder. And just like with the financial crisis, this system will eventually collapse because the underlying reality doesn’t match the representation.

The concept of memetics was first proposed by Richard Dawkins in 1976. In his book The Selfish Gene, Dawkins introduced the idea of a “meme” as a unit of cultural transmission. He defined a meme as “a piece of information that is copied from one individual to another." Dawkins believed that memes were the key to understanding how culture evolves. He argued that memes are responsible for our shared beliefs, values, and behaviors.

Memetics is the study of how information propagates through a population. It is the study of how ideas spread from one person to another. Just like genes, memes are passed from one person to another. And just like genes, memes can mutate and evolve. 

Memes have become a tool to identify the flaws in preexisting symbols. They represent the veracity of cultural and institutional symbols. We are hardwired to pay attention to things that are novel, unexpected, or emotionally arousing. And we are especially susceptible to distractions when we are tired, stressed, or hungry. The degradation caused by weaponized distraction has amplified the effectiveness of memetics in communicating truth.  The power of memetics lies in its ability to bypass our rational defenses and tap into our emotional state. 

The idea of memes has been controversial from the beginning. Some people have embraced it and used it to explain a wide range of phenomena. Others have criticized it for being too reductionist or for being unable to be tested empirically. The error in this criticism is that memetics is a tool to empirically test the accuracy of metaphysical symbols. Epistemology is the study of how we know what we know. It is the study of how we acquire, verify, and justify our beliefs. Memetics is the tool of Epistemics to measure the veracity of metaphysical symbols.

It would make sense that institutions that have undermined their trustability are threatened by memes.  After all, if memes are responsible for identifying the flaws in preexisting symbols, then institutions that have built their power on those symbols are at risk of being exposed. This is why we are seeing a crackdown on meme-makers. Social media platforms are censoring memetic content. Governments are arresting meme-makers. The reason is simple: power is threatened by truth.

### Welcome to Epistemic Warfare

Despite its detractors, the idea of memetics has had a significant impact on our understanding of culture. It has helped us to see that culture is not static, but rather it is constantly evolving. It has also helped us to understand how ideas spread through a population. And finally, it has given us a new way of looking at ourselves and our world. The internet has created a perfect environment for memes to flourish. The speed and reach of the internet allowed memes to spread quickly and widely. And the anonymous nature of the internet allows memes to mutate and evolve without restraint.

The most successful memes are those that can tap into our emotions. They are able to evoke a strong emotional reaction that compels us to share them with others. The best example of this is the “Keep Calm and Carry On” meme. This meme was created in response to the financial crisis of 2008. It taps into our fear and anxiety about the future and turns it into a rallying cry for action. The “Keep Calm and Carry On” meme is an example of a positive reinforcement meme. 

There are also negative reinforcement memes, such as the “Fear Monger” meme. This meme identifies the mechanism to tap into our fears and anxieties in order to sell us products or services that we don’t need. The “Fear Monger” meme is an example of a meme that is being used to exploit our attention. And this is just one example of how our attention is being weaponized. We are under assault from a never-ending stream of distractions. And these distractions are being carefully designed to exploit our cognitive biases. 

While many memes are created to identify less important flaws in cultural untruths, it is possible to weaponize memetics to destroy overarching false narratives. The weaponization of memetics is a new form of epistemic warfare, where the aim is not to win a physical battle, but rather to win the battle of ideas. In epistemic warfare, the goal is to undermine your opponent’s ability to think clearly and make rational decisions. In epistemic warfare, the battlefield is the human mind.

While the war, itself, is as old as time, the proliferation of memetics has enabled the population to arm itself with the truth. The power of memetics lies in its ability to identify and destroy false narratives. By identifying the flaws in our current worldview, memetics can lead us toward a more accurate understanding of reality. In this way, memetics is a powerful tool for correcting a corrupted culture. In ongoing epistemic warfare is imperative to become aware of the tactics being used. Once these tactics are identified, a population can begin to inoculate itself against them. They can begin to create memes of their own that counter the narratives being pushed by those who seek power to control.

As symbols are proven to be inaccurate, they are discarded and new symbols arise to take their place. This is the process of memetic evolution, a process that is constantly happening all around us. Institutions resting on corrupted or untrustworthy symbols will crumble. The only thing that can withstand the power of memetics is the truth. As Frank Herbert said in Dune, the power to destroy a thing is the absolute control over it. In epistemic warfare, the aim is to destroy the false narratives that have taken hold of our culture. And the only way to do this is to replace them with accurate narratives. This is the power of memetics.
